### PR TITLE
Add first project documentation

### PR DESCRIPTION
**Pull request by oleksiibidnichenko**
It is the first version of project documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+
+## Solution architecture
+
+                      ┌────────────────┐
+                      │          perf  │
+                   ┌──┴───────────┐    ├───────────┐
+           TCP     │              │    │           │
+         ┌─────────┤ C++11 based  ├────┘           │
+         │         │  async app   │            ┌───▼────┐
+  ┌──────┴──────┐  └──────────────┘            │        │
```
Comment (netch80) [inline README.md]: Let's specify sides according to TCP (who is client, who is server).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+  │             │                              │ report │
+  │ TCP Traffic │                              │        │
```
Comment (netch80) [inline README.md]: To consider: input data specification (message types, type proportion).

File: README.md [hunk 3]
```diff
@@ -1,2 +1,39 @@
...
+         │         │              │    │
+         └─────────┤ c++17 based  ├────┘
+           TCP     │  async app   │
+                   └──────────────┘
+
+
+### Syntetic net protocol layout
+
+  ┌────────┬────────┬─────────────┬──────────────────────┐
+  │protocol│inverse │payload len. │                      │
```
Comment (netch80) [inline README.md]: Maybe put payload message type here?

**Review by netch80 — CHANGES_REQUESTED** [review]
See inline

File: README.md (hunk 1 above)
Comment (oleksiibidnichenko) [reply README.md]: done by using "*" on the server side

File: README.md (hunk 3 above)
Comment (oleksiibidnichenko) [reply README.md]: protocol version replaced with the "Message Type" field

File: README.md (hunk 2 above)
Comment (oleksiibidnichenko) [reply README.md]: Added database files as inputs for the traffic generator.

**Review by netch80 — APPROVED** [review]

**Review by netch80 — APPROVED** [review]
OK

**Pull request merged**